ChangeLog.md

```diff
@@ -205,6 +207,11 @@ Notable changes to the FreeOrion project will be documented in this file.
 - Fixed issues with AIs replaying their turns when loading a game.
 - Fixed combat log layout / scrollbar issues.
 - Fixed issues with compressed XML saves containing invalid XML.
+- Fixed issue where fleets which had never been moved since their creation were unable to enforce blockades.
+- Fixed segfault scrolling through tabs.
```

Comment:
did this occur in the previous release?

Reply:
You're right, the relevant commit is e59c35a2271278c793e4de9f31057e940683330f (cherry picked as 7f63c1c697b0e9bc70e71d00ba8e0aaf2bbd67e2 in the release branch), which fixed #2133. The function containing the bug (`TabBar::RecalcLeftRightButton()`) has been added last October.

Fixed.